src/integrationTest/java/org/openstreetmap/atlas/generator/slicing/DynamicRelationSlicingTest.java

```diff
@@ -0,0 +1,118 @@
+package org.openstreetmap.atlas.generator.slicing;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.generator.AtlasGeneratorHelper;
+import org.openstreetmap.atlas.generator.tools.caching.HadoopAtlasFileCache;
+import org.openstreetmap.atlas.generator.tools.streaming.ResourceFileSystem;
+import org.openstreetmap.atlas.geography.Polygon;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.items.Line;
+import org.openstreetmap.atlas.geography.atlas.raw.slicing.RawAtlasCountrySlicer;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.geography.sharding.CountryShard;
+import org.openstreetmap.atlas.geography.sharding.DynamicTileSharding;
+import org.openstreetmap.atlas.geography.sharding.Shard;
+import org.openstreetmap.atlas.geography.sharding.Sharding;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+
+/**
+ * @author james-gage
+ */
+public class DynamicRelationSlicingTest
```

Comment:
If this is an integration test, I think we should call it `DynamicRelationSlicingIntegrationTest` to avoid confusion. There are other tests in the atlas repos that are not named this way and I find it confusing!